packages/open-next/src/build/validateConfig.ts

```diff
@@ -0,0 +1,56 @@
+import {
+  FunctionOptions,
+  OpenNextConfig,
+  SplittedFunctionOptions,
+} from "types/open-next";
+
+import logger from "../logger.js";
+
+function validateFunctionOptions(fnOptions: FunctionOptions) {
+  if (fnOptions.runtime === "edge" && fnOptions.experimentalBundledNextServer) {
+    logger.warn(
+      "experimentalBundledNextServer has no effect for edge functions",
+    );
+  }
+  if (
+    fnOptions.override?.generateDockerfile &&
+    fnOptions.override.converter !== "node" &&
+    fnOptions.override.wrapper !== "node"
+  ) {
+    logger.warn(
+      "You've specified generateDockerfile without node converter and wrapper. Without custom converter and wrapper the dockerfile will not work",
```

Comment:
Should this throw an error and abort the build if the dockerfile doesn't work?

Reply:
No because they could use their own implementation here that would show the warning